WalletWasabi.Fluent/Views/Login/LoginView.axaml

```diff
@@ -45,9 +45,13 @@
           <DockPanel DockPanel.Dock="Top"
                      VerticalAlignment="Bottom"
                      IsVisible="{Binding IsPasswordNeeded}">
-            <CopyablePasswordTextBox DockPanel.Dock="Top" Text="{Binding Password}" Watermark="Passphrase" PasswordChar="•" Classes="revealPasswordButton copyablePasswordTextBox">
+            <CopyablePasswordTextBox DockPanel.Dock="Top" Text="{Binding Password}" Watermark="Passphrase" PasswordChar="•" Classes="revealPasswordButton copyablePasswordTextBox"
+                                     x:Name="Password">
               <Interaction.Behaviors>
                 <FocusOnAttachedBehavior />
+                <DataTriggerBehavior Binding="{Binding Password}" Value="">
```

Comment:
It doesn't work when I first press ENTER then click `Ok` on the error message.
The visibility of the DockPanel at L41 might be a better trigger. If true then give the focus to the passwordbox.

Reply:
@soosr Fixed in latest commit